ui.tests/test-module/specs/wizard/wizard.locale.spec.js

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+const afConstants = require("../../libs/commons/formsConstants");
+const sitesSelectors = require("../../libs/commons/sitesSelectors");
+describe('Locale - Authoring Test', function () {
+    if(cy.af.isLatestAddon()) {
+        context('Test Wizard Component String Language', function () {
+            beforeEach(() => {
+                cy.intercept("/editor.html/content/forms/af/language-test.html").as("createFormRequest");
+                cy.intercept("/adobe/forms/fm/v1/themes*").as("getThemesRequest");
```

Comment:
These intercept would fail if request has context path or sling output mappings, please intercept via regex and not URL's

Reply:
Resolved